### ISSUE TITLE
Ability to force the creation of a folder

### ISSUE THREAD

**Issue by giosann** [transmission/transmission#5614]
### What is the issue?

I'd like to be able to create a folder for each torrent i download even if it's a simple file.

The use case would be that i have a folder for movies and I want each movie to have it's own subfolder and as it stands now I have to manually add the folder to the destination path to create it.
The current situation is messy because some movies are only the video file and others include some extras like subs and the like. the movies with only the video file go to the main folder and the movies that already include a folder in the torrent file go into a subfolder as intended. I want all of them to have a subfolder.

Basically I'd like to be able to choose the "Content layout:" like in qBittorrent between "original", "create subfolder" and "don't create subfolder".
This would cover all situations, even if one should not want the folder to be created.

### Which application of Transmission?

None

### Which version of Transmission?

4.0.3

**Comment by killemov** [comment]
And what is your problem with single-file torrents anyway?

**Comment by giosann** [comment]
I might try but i don't think that I would have much better chanches than here since other clients like qBittorrent supoort this.

Anyway I think it would be useful in general to keep the download folder more orderly.

**Comment by Tharn** [comment]
It'd be easier to search the main folder by name when all contents are uniform in subfolders, yes. Seems like a good QoL option.

**Comment by MeirArani** [comment]
Yeah there's plenty of use cases for this. Jellyfin likes to spew all of its metadata files (posters, etc.) into the same folder as video file. If that ends up being just the general `Movies` folder, then you end up with a nightmarish amount of trash clogging the directory. 

I heavily recommend adding this as a simple QoL setting.

**Comment by matru** [comment]
Any updates on this? This is not even a QoL, but a must have for many workflows.
